uelc/main/migrations/0043_auto_20150616_0832.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
If the version with the extra `SectionSubmission` never made it outside your local development environment, it's probably better to just delete this and the migration above (`0042_sectionsubmission.py`) and pretend they never happened.